_security-analytics/usage/findings.md

```diff
@@ -37,12 +37,31 @@ As one more alternative, you can select an option from the **Recently used date
 
 ## The Findings list
 
-The Findings list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.
+The **Findings** list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.
 <br><img src="{{site.url}}{{site.baseurl}}/images/Security/finding-list.png" alt="A list of all findings" width="85%">
```

Comment:
The text following this image is displayed on the same line as the image because there is no new line between them. But this is existing documentation, so I would address it in another PR. Simply remove the `<br>` and put the image on line 42, and insert a new line after the image, so the "Use the..." text starts at line 44.

Reply:
Agreed. Will need to clean these up in another pass.